index.html

```diff
@@ -208,5 +208,31 @@ <h2>Skills</h2>
         </ul>
     </section>
   </main>
+  <footer id="footer-contact">
+    <section id="contact-form">
+      <p id="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <form action="https://formspree.io/f/mnqreoov" method="post">
+        <div id="input-name" class="input-wrapper">
+          <input type="text" required  maxlength="30" placeholder="Full Name">
+        </div>
+        <div id="input-email" class="input-wrapper">
+          <input type="email" name="email" required placeholder="Email Adress">
+        </div>
+        <textarea required name="message" id="" cols="30" rows="10" maxlength="500" placeholder="Enter text here"></textarea>
+        <center><button class="btn" type="submit" id="action">Get in touch</button></center>
+      </form>
+    </section>
+    <section id="social-links">
+      <div class="social-icons">
+        <img src="img/Github.svg" alt="Github icon">
+        <img src="img/LinkedIn.svg" alt="Linkedin icon">
+        <img src="img/Hand.svg" alt="Hand icon">
+        <img src="img/Twitter.svg" alt="Twitter icon">
+        <img src="img/M.svg" alt="Medium icon">
+      </div>
```

Comment:
- [ ] Good job adding the `alt` attributes on the images. Kindly wrap the images in an `a` tag as suggested by the previous reviewer. It'll make it easier for people to reach you if you add your social links in the `href` attribute of the link. ✍️
- [ ] [OPTIONAL] Since we are putting a list of links here, I think it would be more appropriate to structure them using `ul` and `li`.

**Example**
```
<ul class="social-icons">
  <li><a href="#"><img src="img/Github.svg" alt="Github icon"></a></li>
  <li><a href="#"><img src="img/LinkedIn.svg" alt="Linkedin icon"></a></li>
</ul>
```

Reply:
It is corrected as advised by code reviewer